Src/App/Controllers/postController.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable no-console */
+const createPost = (req, res) => {
+  console.log(req.body);
```

Comment:
Avoid pushing console.log code , you should be using process.stdout.write(`req.body`); if you want to push a console.log 

So use console.log only for local developments